README.md

```diff
@@ -2,27 +2,47 @@
 
 Use System Initiative workspaces in your GitHub workflows!
 
-NOTE: this is in early private access, and not yet publicly available.
+You can use this GitHub Action to take the following actions:
 
-## Trigger
+1. Update a component to set properties and subscriptions
+2. Trigger a management function attached to a component
 
-Triggers a
-[management function](https://www.systeminit.com/blog/announcing-management-functions),
-so you can deploy real infrastructure based on your changes.
+## Usage
 
-### Usage
+### Update Component
 
-Place this in a `.github/workflows` workflow on your repository:
+```yaml
+- uses: systeminit/actions@v1
+  with:
+    changeSetName: CI
+    component: testing-component
+    attributes: |
```

Comment:
You don't *have* to, but I would strive to use yaml where you can in examples; it's a much simpler model for the user (and kinda nice).